### PR TITLE
add instruction in readme.md

### LINKED ISSUE #177: Documentation : Project Setup Documentation Update
## Issue Summary

A significant number of contributors have reported facing challenges when setting up the project . These challenges often lead to delays, confusion, and difficulty getting the project running smoothly. 

## Updates
This document provides a step-by-step guide to setting up the project, including detailed instructions for users setting it up in WSL. We have also included solutions to common errors encountered during setup, along with best practices to ensure a smooth and successful installation.

### PR DESCRIPTION
**Pull request by Rajgupta36**
Resolves #177 

added instruction .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,14 +7,14 @@ Follow these steps to set up the OWASP Nest application:
 1. **Fork the Repository**:
    - Fork <https://github.com/OWASP/Nest> repository using "Fork" button
 
-1. **Clone the Repository**:
+2. **Clone the Repository**:
```
Comment (arkid15r) [inline README.md]: why did you change these?

File: README.md [hunk 2]
```diff
@@ -201,6 +201,7 @@ This should resolve any permission-related issues and ensure the frontend depend
    - Required order:
      1. First terminal: `make run` (wait until it's fully started)
      2. Second terminal: `another_cmds`
-   - Note: Keep all terminals open while working.
-5. **NPM conflict error**
-   - Do not mix PowerShell npm commands with WSL npm commands in the same project. Use only one environment consistently to avoid conflicts.
+---
```
Comment (arkid15r) [inline README.md:207]: I got the point. Let's merge it.

File: README.md (hunk 1 above)
Comment (Rajgupta36) [reply README.md:10]: i manually hardcoded the values. is it bad ?

Comment (Rajgupta36) [reply README.md]: > why did you change these?

i manually hardcoded the values instead of autoformatting. is it bad ?

Comment (arkid15r) [reply README.md]: auto- is easier to modify

**Review by arkid15r — APPROVED** [review]
